### ISSUE TITLE
Crash when not find config file

### ISSUE THREAD

**Issue by qarmin** [iphands/randy#25]
I think that randy should load default(builtin) config when not found any.
It is strange that after installation we need to create/copy config file because it crash when we don't do it
```
thread 'main' panicked at 'Unable to open/read ./config/default.yml', src/main.rs:661:19
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Przerwane (zrzut pamięci)
```

**Comment by iphands** [comment]
Yeah I think Im getting closer and closer to that... there are now quite a few defaults that are assumed... but still you *must* include at least one config file.

FWIW when packaged I was putting the config file in a known place like /usr/share/randy/config
See: https://github.com/iphands/randy/blob/main/packaging/gentoo/randy.ebuild#L131

So *if* Randy is packaged this should be a non issue even though the config is externalized.

**Comment by ssokolow** [comment]
What I do with my own creations is embed the default config file using `include_str!` which can be written out using an option like `--write-conf <path>`... of course, I tend to want a default config that I can then use in a fallback chain via [figment](https://lib.rs/crates/figment).

Either way, not a good idea to crash rather than displaying a clean failure message. When I saw the crash, my eyes glossed over the reason for the failure as I rushed to re-run it with `RUST_BACKTRACE` to figure out where the bug was.

**Comment by iphands** [comment]
+1 on both accounts!
